Content.Server/DeltaV/Implants/SyrinxImplantSystem.cs

```diff
@@ -1,115 +1,30 @@
-using Content.Server.Administration.Logs;
-using Content.Server.Implants;
-using Content.Server.Speech.Components;
-using Content.Shared.Database;
-using Content.Shared.DeltaV.Implants;
+using Content.Server.VoiceMask;
 using Content.Shared.Implants;
-using Content.Shared.Inventory;
-using Content.Shared.Popups;
-using Content.Shared.Preferences;
-using Content.Shared.Speech;
-using Content.Shared.VoiceMask;
-using Robust.Shared.Prototypes;
+using Content.Shared.Tag;
 
-namespace Content.Server.DeltaV.Implants;
+namespace Content.Server.Implants;
 
-public sealed class SyrinxImplantSystem : EntitySystem
+public sealed class SubdermalBionicSyrinxImplantSystem : EntitySystem
 {
-    [Dependency] private readonly IAdminLogManager _adminLogger = default!;
-    [Dependency] private readonly IPrototypeManager _proto = default!;
-    [Dependency] private readonly SharedPopupSystem _popup = default!;
-    [Dependency] private readonly SharedUserInterfaceSystem _ui = default!;
+    [Dependency] private readonly TagSystem _tag = default!;
+
+    [ValidatePrototypeId<TagPrototype>]
+    private const string BionicSyrinxImplant = "BionicSyrinxImplant";
```

Comment:
no reason to have this tag check, if you dont want a voice mask implant to work dont make one?

Reply:
Very good point...